src/components/BooksList.js

```diff
@@ -16,6 +16,10 @@ const BookList = () => {
   // const listBooks = useSelector((store) => {
   //   return store.books.array;
   // });
```

Comment:
- [ ] Same thing here, please check the above ⬆

Reply:
Thank you so much that is very good obervation.